backend/src/errors/internal.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This class pertains to any internal errors that may not occur
+ */
+
+import { CustomError } from "./errors";
+
+const ERROR_GETTING_MENTEE = "There was some error getting the mentee";
+const ERROR_GETTING_MENTOR = "There was some error getting the mentor";
+const ERROR_GETTING_IMAGE = "There was some error getting the image";
+const NO_APP_PORT = "Could not find app port env variable";
+const NO_MONGO_URI = "Could not find mongo uri env variable";
+const NO_SERVICE_ACCOUNT_KEY = "Could not find service account key env variable";
+const NO_DEFAULT_IMAGE_ID = "Could not find default image id env variable";
+
+export class InternalError extends CustomError {
```

Comment:
Let's use 500 - Internal Server Error for these. 
403 stands for Forbidden which does not apply here